packages/alfa-rules/src/common/outcome/contrast.ts

```diff
@@ -0,0 +1,32 @@
+import { Contrast } from "../diagnostic/contrast";
+import { Err, Ok } from "@siteimprove/alfa-result";
+
+export namespace Outcomes {
```

Comment:
It would better be named `Contrast`, but that is already used for the `Diagnostic` class, and since these two are almost always imported together, that need renaming. And if we rename something every time we import it, it probably means that the name isn't great…

Still, not happy of having this just called `Outcomes` 😞

Reply:
This should do the trick:

```ts
import { Contrast as ContrastDiagnostic } from "../diagnostic/contrast";
```

Reply:
Yes, of course.
What stopped me is that this will be needed at the 3 places where we import that (here and the two test files).
If we have to rename it every time we import it, it feels like the name was wrong in the first place 😕

Reply:
> If we have to rename it every time we import it, it feels like the name was wrong in the first place 😕 

The beauty of modules is that they don't need to concern themselves with the world around them. They should choose the best possible export names _taking only their own context into account_. Importing modules can always rename individual imports or even namespace entire imported modules to avoid collisions _in their own context_.

"Effective Go" has a similar philosophy on package and export names: https://golang.org/doc/effective_go.html#package-names